public/emoji/activity.css

```diff
@@ -0,0 +1,1933 @@
+/* glue: 0.13 hash: efcf7f4b00 */
+        .em-activity-26f9,
```

Comment:
Nit: Indentation intentional?

Reply:
This is a generated file :slightly_smiling_face: